src/main/groovy/org/shipkit/internal/gradle/configuration/DeferredConfiguration.java

```diff
@@ -45,4 +51,11 @@ public void execute(Project project) {
             }
         });
     }
+
+    @ExposedForTesting
+    public static void forceConfiguration(Project project){
```

Comment:
So far, we have been using ```project.evaluate()``` to simulate this. Why do we need forceConfiguration?